diff -burN webmail.orig/badlogin.php webmail/badlogin.php
--- webmail.orig/badlogin.php	2006-07-16 21:57:53.000000000 +0200
+++ webmail/badlogin.php	2006-07-16 21:59:17.000000000 +0200
@@ -21,6 +21,8 @@
 
 $smarty = new Smarty;
 $smarty->compile_dir = $temporary_directory;
+$smarty->template_dir = 'themes';
+$smarty->config_dir = 'langs';
 $smarty->security=true;
 $smarty->secure_dir=array("./");
 
diff -burN webmail.orig/error.php webmail/error.php
--- webmail.orig/error.php	2006-07-16 21:57:53.000000000 +0200
+++ webmail/error.php	2006-07-16 21:59:26.000000000 +0200
@@ -19,6 +19,8 @@
 require_once(SMARTY_DIR."Smarty.class.php");
 $smarty = new Smarty;
 $smarty->compile_dir = $temporary_directory;
+$smarty->template_dir = 'themes';
+$smarty->config_dir = 'langs';
 $smarty->security=true;
 $smarty->secure_dir=array("./");
 $smarty->assign("umLanguageFile",$selected_language.".txt");
diff -burN webmail.orig/inc/inc.php webmail/inc/inc.php
--- webmail.orig/inc/inc.php	2006-07-16 21:57:53.000000000 +0200
+++ webmail/inc/inc.php	2006-07-16 21:58:47.000000000 +0200
@@ -24,6 +24,8 @@
 require_once(SMARTY_DIR."Smarty.class.php");
 $smarty = new Smarty;
 $smarty->compile_dir = $temporary_directory;
+$smarty->template_dir = 'themes';
+$smarty->config_dir = 'langs';
 $smarty->security=true;
 //$smarty->secure_dir=array("./");
 
diff -burN webmail.orig/index.php webmail/index.php
--- webmail.orig/index.php	2006-07-16 21:57:53.000000000 +0200
+++ webmail/index.php	2006-07-16 21:59:04.000000000 +0200
@@ -22,6 +22,8 @@
 $smarty->security=true;
 $smarty->secure_dir=array("./");
 $smarty->compile_dir = $temporary_directory;
+$smarty->template_dir = 'themes';
+$smarty->config_dir = 'langs';
 $smarty->assign("umLanguageFile",$selected_language.".txt");
 
 $smarty->assign("umEmail",$f_email);
