diff -Nur kopia/download.php new/download.php
--- kopia/download.php	Fri Aug 29 23:42:52 2003
+++ new/download.php	Sat Aug 30 23:30:42 2003
@@ -9,7 +9,8 @@
 S?o Paulo - Brasil
 *************************************************************************/
 
-
+ini_set("session.cache_limiter","public");
+$forcecache = "yes";
 // load session management
 require("./inc/inc.php");
 // check for all parameters
diff -Nur kopia/inc/class.uebimiau.php new/inc/class.uebimiau.php
--- kopia/inc/class.uebimiau.php	Fri Aug 29 23:42:52 2003
+++ new/inc/class.uebimiau.php	Sat Aug 30 23:39:12 2003
@@ -151,7 +151,8 @@
 	function convert_charset($string, $from, $to) {
 		$string = htmlentities($string, ENT_COMPAT, $from);
 		if(function_exists('html_entity_decode')) { //PHP 4.3+
-			return html_entity_decode($string, ENT_COMPAT, $to);
+			$string = html_entity_decode($string, ENT_COMPAT, $to);
+			return $string;
 		} else {
 			$trans_tbl = get_html_translation_table (HTML_ENTITIES);
 			$trans_tbl = array_flip ($trans_tbl);
diff -Nur kopia/inc/inc.php new/inc/inc.php
--- kopia/inc/inc.php	Fri Aug 29 23:42:52 2003
+++ new/inc/inc.php	Sat Aug 30 23:29:38 2003
@@ -148,6 +148,8 @@
 /*
 Don't remove the fallowing lines, or you will be problems with browser's cache 
 */
+if($forcecache != "yes")
+{
 Header("Expires: Wed, 11 Nov 1998 11:11:11 GMT\r\n".
 "Cache-Control: no-cache\r\n".
 "Cache-Control: must-revalidate\r\n".
@@ -157,6 +159,7 @@
 <META HTTP-EQUIV=\"Cache-Control\" CONTENT=\"no-cache\">
 <META HTTP-EQUIV=\"Expires\" CONTENT=\"-1\">
 <META HTTP-EQUIV=\"Pragma\" CONTENT=\"no-cache\">";
+}
 // Sort rules
 
 
